### PR TITLE
Update to new slide deck

### PR DESCRIPTION
**Pull request by romanutti**
Demo can be found [here 👀](https://romanutti.github.io/zuehlke-reveal-demo/#/)\*

\* both repos should be transferred to https://github.com/Zuehlke

### CODE REVIEW

**Comment by r3dDoX** [comment]
How do you feel about a dark code theme?
I'd like the clear box it provides, for example, "GitHub Dark Dimmed" would be a nice one. The playground is here: https://highlightjs.org/static/demo/

**Comment by r3dDoX** [comment]
Bildschirmfoto 2022-09-06 um 09 14 56

**Comment by romanutti** [comment]
> How do you feel about a dark code theme? I'd like the clear box it provides, for example, "GitHub Dark Dimmed" would be a nice one. The playground is here: https://highlightjs.org/static/demo/

Like it, especially that the colors are close to the Zühlke colors

**Review by culas — APPROVED** [review]
I tried this version on the CSS course (only the basics chapter) and it works quite well out-of-the-box without adjustments. I like the changes, looks great.
Thanks Marco for the big bulk of the work ❤️

File: src/styles/zuehlke-toc.css [hunk 1]
```diff
@@ -1,88 +0,0 @@
-
```
Comment (culas) [inline src/styles/zuehlke-toc.css:1]: I just realised we need these styles, so they need to be updated too. Most courses use this as overview page linking to the chapters.

Comment (romanutti) [reply src/styles/zuehlke-toc.css:1]: Design-proposal to have it aligned with the new slide-deck:

zuehlke-toc-slide


What do you think?

**Review by mariokaufmann — APPROVED** [review]
Thanks @romanutti for all the work!

File: src/styles/zuehlke-toc.css (hunk 1 above)
Comment (romanutti) [reply src/styles/zuehlke-toc.css:1]: Discussed offline, will be handled in a separate PR

**Pull request merged**